### PR TITLE
[april fools testmerge] The SM turned itself into a pickle

### PR DESCRIPTION
**Pull request by fluffe9911**
🆑 Fluffe9911
add: Our sensors show the SM has turned itself into a pickle, funniest shit we ever seen.
/🆑
Untitled

### CODE REVIEW

File: code/modules/power/supermatter/supermatter.dm [hunk 1]
```diff
@@ -789,8 +789,8 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 	is_main_engine = TRUE
 
 /obj/machinery/power/supermatter_crystal/shard
-	name = "supermatter shard"
-	desc = "A strangely translucent and iridescent crystal that looks like it used to be part of a larger structure."
+	name = "Pickle Super Matter Shard"
+	desc = "Second funniest shit I’ve ever seen"
```
Comment (tralezab) [inline code/modules/power/supermatter/supermatter.dm]: please add a period

**Comment by LemonInTheDark** [comment]
Based as fuck.

**Comment by Fikou** [comment]
>not calling it picklematter

**Comment by trollbreeder** [comment]
not funny didnt laugh

**Comment by nicbn** [comment]
Pls Make everyone in a department radius start laughing uncrontollaby if it starts overheating

**Comment by HomocidlePancakes** [comment]
Needs sounds.

**Comment by necromanceranne** [comment]
pickle slice? hello?

**Comment by cacogen** [comment]
>, funniest shit we ever seen.

get rid of that or just don't changelog it at all

**Pull request closed without merging**